scripts/rspec/rspec.ps1

```diff
@@ -243,44 +243,41 @@ function RemoveText($textToRemove, $modifiableString)
 }
 
 function AppendVbTestCase($ruleTestsFolder) {
-    $existingClassText = Get-Content -Path "${ruleTestsFolder}\\${csClassName}Test.cs" -Raw
-    $snippetText = Get-Content -Path "${RuleTemplateFolder}\\VbNetTestSnippet.cs" -Raw
-
     $usingToken = "using SonarAnalyzer.Rules.CSharp;"
-    $csUsingToken = "using CS = SonarAnalyzer.Rules.CSharp;"
-    $vbNetUsingToken = "using VB = SonarAnalyzer.Rules.VisualBasic;"
+    $usingTokenCS = "using CS = SonarAnalyzer.Rules.CSharp;"
+    $usingTokenVB = "using VB = SonarAnalyzer.Rules.VisualBasic;"
     $namespaceToken = "namespace SonarAnalyzer.UnitTest.Rules"
     $token = "    }"
-    $usingTokenIdx = $existingClassText.IndexOf($usingToken)
 
-    $newText = $existingClassText
+    $text = Get-Content -Path "${ruleTestsFolder}\\${csClassName}Test.cs" -Raw
+    $methodVB = Get-Content -Path "${RuleTemplateFolder}\\TestMethod.VB.cs" -Raw
+    $methodVB = $methodVB.Replace("<", "<VB.")
```

Comment:
We know that `cs` part already exists and we are appending the `vb` part. So instead of this replace, why don't we already have <VB. in the `TestMethod.VB.cs` file?

Reply:
Indeed, good catch! I was under impression that the snippet is used inside the `TestVB.cs` too